src/loaders.py

```diff
@@ -86,16 +92,22 @@ def collate_fn(self, batch):
           images, stacked bbox_targets, stacked clf_targets.
         """
         imgs = [x[0] for x in batch]
-        boxes = [x[1] for x in batch]
-        labels = [x[2] for x in batch]
+        boxes = [x[1][0] for x in batch]
+        labels = [x[1][1] for x in batch]
 
         inputs = torch.stack(imgs)
+        input_size = torch.Tensor(list(inputs.size()[2:]))
         bbox_targets, clf_targets = [], []
         for i in range(len(imgs)):
```

Comment:
@taraspiotr @apyskir whenever I see (or write) `for i in range(len(imgs)):` I feel like a little puppy somewhere is crying but the best alternative I can think of is merely  `for i, img in enumerate(imgs):` or `for i, _ in enumerate(imgs):` . Is it better? Maybe it feels just a tad nicer I guess

Reply:
@taraspiotr @jakubczakon Maybe just `for box, label in zip(boxes, labels):`? It should work, right? And is nice and clean

Reply:
@apyskir definitely